src/project-css/app/Http/Controllers/TableController.php

```diff
@@ -604,7 +604,7 @@ public function prepareLine($curLine, $delimiter, $orgCount, $prcssd) {
           // save the last row position and the Tokenized row
           $this->lastErrRow = $prcssd;
           $this->savedTkns = $tkns;
-          return (false);
+          return (NULL);
```

Comment:
why are you returning null instead of false?  Null can give false positives in some instances.

Reply:
changed.